File: app/src/main/java/com/chess/cryptobot/model/response/livecoin/LivecoinBalanceResponse.java
package com.chess.cryptobot.model.response.livecoin;

import com.chess.cryptobot.model.response.BalanceResponse;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LivecoinBalanceResponse extends LivecoinResponse implements BalanceResponse {
    @SerializedName("value")
    @Expose
    private Double value;

    @Override
    public Double getAmount() {
        return value;
    }
}

File: app/src/main/java/com/chess/cryptobot/model/response/livecoin/LivecoinBalanceResponse.kt
package com.chess.cryptobot.model.response.livecoin

import com.chess.cryptobot.model.response.BalanceResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LivecoinBalanceResponse : LivecoinResponse(), BalanceResponse {
    @SerializedName("value")
    @Expose
    override val amount: Double? = null
}